{{cookiecutter.github_repository}}/.travis.yml

```diff
@@ -23,6 +28,9 @@ install:
   - pip install -r requirements/development.txt
 
 before_script:
+- sudo sed -i 's/port = 5433/port = 5432/' /etc/postgresql/10/main/postgresql.conf
+- sudo cp /etc/postgresql/{9.4,10}/main/pg_hba.conf
+- sudo service postgresql restart
```

Comment:
@sanyam-fueled is there any documentation on why this is required now? is the default port changed in pg10?

Also, just for my clarity, what does `sudo cp /etc/postgresql/{9.4,10}/main/pg_hba.conf` supposed do to?

Reply:
This is for enabling Postgres 10 on travis as we discussed in the call, but I'll see if there is another way and if travis supports it directly now.

Reply:
> @sanyam-fueled is there any documentation on why this is required now? is the default port changed in pg10?
> 
> Also, just for my clarity, what does `sudo cp /etc/postgresql/{9.4,10}/main/pg_hba.conf` supposed do to?

+1. I have seen that installing the package enables it to. Is that not happening here ? Is postgres 9.4 installed by default ? I dont see the 9 version being installed and yet we are copying over the conf.

Reply:
The default postgres which is 9.4 is installed on travis. For the version we specified, Postgres 10, it then installs it, but since the port `5432` (default fo postgres) is already taken by Postgres 9, Postgres 10 takes the next available port i.e. `5433`.

So, we ensure that our system is tested on Postgres 10 and not on Postgres 9.

I hope it makes sense. Please let me know if you've any more questions.